src/main/kotlin/com/openlattice/collections/CollectionTemplateType.kt

```diff
@@ -0,0 +1,47 @@
+package com.openlattice.collections
+
+
+import com.fasterxml.jackson.annotation.JsonCreator
+import com.fasterxml.jackson.annotation.JsonProperty
+import com.openlattice.client.serialization.SerializationConstants
+
+import java.util.Optional
+import java.util.UUID
+
+data class CollectionTemplateType(
```

Comment:
If you're going to have id, name, title, description consider making it some sort of securable object, especially if on the backend you are using the the Reservation service to assign ids.